index.js

```diff
@@ -1,7 +0,0 @@
-if (process.env.NODE_ENV !== 'production') {
```

Comment:
This _use_ to be the main server file that allowed the code to use es6 syntax before running. This is no longer needed. Nextjs picks up `.env` files automatically.